xbmc/epg/EpgInfoTag.h

```diff
@@ -454,6 +464,8 @@ namespace EPG
 
     CEpg *                   m_epg;                /*!< the schedule that this event belongs to */
 
+    unsigned int             m_iFlags;             /*!< the status flags applicable to this EPG entry.  More bitflags will be added in a future release */
```

Comment:
Please remove the comment re "future release". I changed my mind, we don't need this imo (at least not here). :-/